src/test/groovy/graphql/nadel/result/ResultComplexityAggregatorTest.groovy

```diff
@@ -63,7 +65,7 @@ class ResultComplexityAggregatorTest extends Specification {
         resultComplexityAggregator.incrementServiceNodeCount("service2", 5)
 
         then:
-        resultComplexityAggregator.toString() == "ResultComplexityAggregator{totalNodeCount=24, serviceNodeCountsMap={service1=14, service2=10}}"
+        resultComplexityAggregator.toString() == "ResultComplexityAggregator{totalNodeCount=24, serviceNodeCountsMap={service2=10, service1=14}}"
```

Comment:
why did this change?

Reply:
I think this comes from the fact that we are not using LinkedHashMap anymore and this also means this can change randomly.
I think we should use `LinkedHashMap` again but with a synchronized Wrapper (Collections.synchronizedMap). This should fix it.

Reply:
yea I am just investigating this now, in any case I don't think the ordering of the elements matter, so i can change the toString test to not care about ordering